pkg/appmanager/resourceConfig.go

```diff
@@ -257,7 +257,11 @@ func getRouteServices(route *routeapi.Route) []RouteService {
 	if route.Spec.AlternateBackends != nil {
 		for _, svc := range route.Spec.AlternateBackends {
 			svcs[svcIndex].name = svc.Name
-			svcs[svcIndex].weight = int(*(svc.Weight))
+			if svc.Weight != nil {
```

Comment:
This is not possible in a real Openshift environment. Is this needed just for unit tests?

Reply:
If it's not possible in OpenShift, then I'll remove it and make sure the tests supply a weight.

Reply:
It's not possible.